buddyboss/buddyboss-platform
# PROD-781 Symbolic links settings changed when updating the activity settings - issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2647  .

### How to test the changes in this Pull Request:

1. Go to media settings and disable the Symbolic link option (if enabled)
2. Move on the activity setting page and just click save settings
3. Again back to the media settings and check the Symbolic link option
4. You will see the option is not enabled

### Proof Screenshots or Video

https://www.loom.com/share/6518689706da43a18436f801ac5b194b

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
